src/main/java/openblocks/common/entity/EntityHangGlider.java

```diff
@@ -171,6 +272,41 @@ private void fixPositions(EntityPlayer thePlayer, boolean localPlayer) {
 		this.motionZ = this.posZ - this.prevPosZ;
 	}
 
+	private boolean isLocalPlayer() {
+		return worldObj.isRemote && player == Minecraft.getMinecraft().thePlayer;
```

Comment:
Please use `OpenMods.proxy.isClientPlayer`. Using `Minecraft` class in code executed on server in risky.